### PR TITLE
Burger menu

### PR DESCRIPTION
**Pull request by jfidiles**

### CODE REVIEW

**Comment by jfidiles** [comment]
Changes done

File: _sass/main/_body.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.Body-dark {
```
Comment (artisologic) [inline _sass/main/_body.scss]: --dark

File: _sass/main/_body.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  @include dark-container-colors();
+}
+
+.Body-light {
```
Comment (artisologic) [inline _sass/main/_body.scss]: --light

**Comment by jfidiles** [comment]
Done @artisologic !

File: _includes/header.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
         {% else %}
             {% assign header-type = 'dark' %}
         {% endif %}
-        <a href="{{ site.url }}">
+        <a href="{{ "/" relative_url }}">
```
Comment (artisologic) [inline _includes/header.html]: forgot the pipe here

**Pull request merged**